src/views/Farms/components/FarmTable/Liquidity.tsx

```diff
@@ -13,41 +15,42 @@ const LiquidityWrapper = styled.div`
   min-width: 110px;
   font-weight: 600;
   text-align: right;
+  margin-right: 14px;
 
-  ${({ theme }) => theme.mediaQueries.sm} {
+  ${({ theme }) => theme.mediaQueries.lg} {
     text-align: left;
+    margin-right: 0;
   }
 `
 
 const Container = styled.div`
   display: flex;
   align-items: center;
-
-  svg {
-    margin-left: 14px;
-  }
-
-  ${({ theme }) => theme.mediaQueries.sm} {
-    svg {
-      margin-left: 0;
-    }
-  }
 `
 
 const Liquidity: React.FunctionComponent<LiquidityProps> = ({ liquidity }) => {
   const displayLiquidity = liquidity
     ? `$${Number(liquidity).toLocaleString(undefined, { maximumFractionDigits: 0 })}`
     : '-'
   const TranslateString = useI18n()
+  const { targetRef, tooltip, tooltipVisible } = useTooltip(
+    TranslateString(999, 'The total value of the funds in this farm’s liquidity pool'),
+    'top-end',
+    'hover',
+    undefined,
+    undefined,
+    [20, 10],
+  )
```

Comment:
@Chef-Cheems Now that I see this in the wild I think it best to make the hooks second parameter an object. It will be much cleaner than this.

```
useTooltip('string', {
  placement: 'top-end',
  trigger: 'hover',
  tooltipOffset: [20,10]
})
```

What do you think?

Reply:
@Chef-Cheems yeah, agree. I started to think the same too. Will do it this week. 👍